appserver/connectors/connectors-runtime/src/main/java/com/sun/enterprise/connectors/ConnectorConnectionPool.java

```diff
@@ -58,7 +58,6 @@
  *
  * @author Srikanth Padakandla
  */
-
```

Comment:
More deletion of random whitespace, no need for a line between the Javadoc and the first line of the class - looks messy and isn't in keeping with the format of other classes.